### PR TITLE
Update language in docs for animation player & debugging

### PR DESCRIPTION
**Pull request by devjarmo**
As I use Flax I'd like to contribute to the documentation. These are items that were unclear to me so I've attempted to add clarifying language.

### CODE REVIEW

**Comment by mafiesto4** [comment]
Thanks!

File: manual/scripting/debugging/visual-studio.md [hunk 1]
```diff
@@ -31,10 +31,12 @@ Click on a left side of the code editor to assign a breakpoint to the line. Red
 
 ### 5. Attach to Flax
 
-Press the `Attach to Flax` button with a green arrow to connect Visual Studio to Flax Editor.
+Press the `Attach to Flax` button with a green arrow to connect Visual Studio to Flax Editor. Visual Studio should enter a debugging mode and the bottom of the IDE will remain orange. This signifies Visual Studio is ready for debugging. If the bottom of the IDE does not remain orange, there is an issue with Flax and VS integration.
```
Comment (stefnotch) [inline manual/scripting/debugging/visual-studio.md]: This feels a bit wordy. Maybe cut out the middle sentence?

File: manual/animation/scene-animations/tutorials/animate-ui.md [hunk 2]
```diff
@@ -36,6 +36,6 @@ Property track editing depends on the value type. For colors and vectors, you ca
 
 ### 5. Play the animation
 
-The final step is to add Scene Animation Player actor and play this animation.
+The final step is to add Scene Animation Player actor to the scene. After adding the Animation Player to the scene you can then select the Scene Animation player in the Animation Player window. Next run the game in Flax Editor by selecting play, and then hit play on the Animation Player window to preview your animation.
```
Comment (stefnotch) [inline manual/animation/scene-animations/tutorials/animate-ui.md]: That's some pretty useful information! Though, is there any way to rephrase this to not repeat "Animation Player" so often?
